app/controllers/authorities_controller.rb

```diff
@@ -20,13 +20,20 @@ def get_results(subject, search)
     lcsh_base_uri = "http://id.loc.gov/authorities/subjects/suggest/?q="
     # can increase limit to max 20 with '&rows=20'
     fast_geo_uri = "http://fast.oclc.org/searchfast/fastsuggest?fl=suggest51&wt=json&query="
+    
     if subject == 'lcsh'
       result = JSON.parse(HTTParty.get(
         lcsh_base_uri + "#{search.gsub(/\s/,'*')}*"
-      ))[1]
+      ))
+      result = result[1] ? result[1] : ["Error Searching"]
```

Comment:
So `result` is guaranteed to be an array?

Reply:
it is now, right? it should default to `["Error Searching"]`, no?